tools/schematics/testing.ts

```diff
@@ -5,6 +5,39 @@ import glob from 'glob';
 import path from 'path';
 import semver from 'semver';
 
+const featureLibsFolders: string[] = [
+  'asm',
+  'cart',
+  'organization',
+  'product',
+  'product-configurator',
+  'qualtrics',
+  'smartedit',
+  'storefinder',
+  'tracking',
+  'user',
+];
+const commands = [
```

Comment:
If you want to go even further we can generate part of this list based on `featureLibsFolders` and treat this cases in `default` in switch statement later to avoid listing all cases ;)

Reply:
Discussed offline - we can give it a go in the future, especially with the new TS.